//package core.io.protocol;

message CEffectData
{
	optional int32  type = 1;//效果类型
	optional int32	param = 2;//参数
}

message BattleOneUnit
{
	optional string 	id        		= 1;//式神ID
	optional int32		role			= 2;//1攻击方 2防守方
	optional int32 	  	hp            	= 3;//剩余血量
	optional int32 	  	maxhp        	= 4;//总血量
	optional int32		furypoint		= 5;//当前怒气
	optional int32		slideacctime 	= 6;//滑动技能累加时间
	repeated CEffectData effectlist		= 7;//效果列表
}

